core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -1958,3 +1958,16 @@ class TaskSetManagerSuite
     }
   }
 }
+
+class FakeLongTasks(
+    stageId: Int,
+    partitionId: Int)
+    extends FakeTask(stageId, partitionId) {
```

Comment:
indentation?

Reply:
ran scalafmt and updated

Reply:
Got it. Actually, I meant only line 1965.